x-pack/plugins/observability_solution/synthetics/server/synthetics_service/formatters/formatting_utils.ts

```diff
@@ -51,7 +51,7 @@ export const replaceStringWithParams = (
 };
 
 export const hasNoParams = (strVal: string) => {
-  const shellParamsRegex = /\$\{[a-zA-Z_][a-zA-Z0-9_]*\}/g;
+  const shellParamsRegex = /\$\{[a-zA-Z_][a-zA-Z0-9\._\-?:]*\}/g;
```

Comment:
No a big deal, but you could move this var definition outside the function.

Reply:
Nice idea, I will implement that.

Reply:
[2a02e32](https://github.com/elastic/kibana/pull/178054/commits/2a02e32087086b215473376995eab1bc7ee69e30)